network-store-iidm-impl/src/main/java/com/powsybl/network/store/iidm/impl/NetworkImpl.java

```diff
@@ -103,7 +103,7 @@ class BusBreakerViewImpl implements BusBreakerView {
 
         @Override
         public Iterable<Bus> getBuses() {
-            return getBusStream().collect(Collectors.toList());
+            return getBusStream().toList();
```

Comment:
Stream.toList() returns an unmodifiiable list while Stream.collect(Collectors.toList()) returns a modifiable list. Are we sure those collections are not modified somewhere in our code ?

Reply:
Not sure at all ! Ok I remove all of this, let's be careful.